src/renderer/features/proxies/model/__tests__/proxies-model.test.ts

```diff
@@ -67,16 +68,34 @@ describe('entities/proxy/model/proxy-model', () => {
 
   test('should add proxy ', async () => {
     const scope = fork({
-      values: new Map().set(networkModel.$chains, {
-        '0x01': {
-          chainId: '0x01',
-          name: 'Westend',
-          options: ['regular_proxy'],
-        },
-      }),
+      values: new Map()
+        .set(networkModel.$chains, {
+          '0x01': {
+            chainId: '0x01',
+            name: 'Westend',
+            options: ['regular_proxy'],
+          },
+        })
+        .set(walletModel.$accounts, [
```

Comment:
Let's use one-liners for objects with 1-3 values, otherwise we occupy so much unnecessary space 😔
`.set(walletModel.$accounts, [{ walletId: 1, accountId: '0x01' }])`